app/assets/javascripts/controllers/live_metrics/util/metrics-utils-factory.js

```diff
@@ -96,41 +96,59 @@ angular.module('miq.util').factory('metricsUtilsFactory', function() {
       dash.chartDataInit = true;
     }
 
-    var getContainerDashboardData = function(item, response) {
-      'use strict';
-      if (checkResponse(response) === false) {
-        return;
+    var timeTooltip = function (data) {
+      return '<div class="tooltip-inner">' +
+        moment(data[0].x).format('MM/DD hh:mm') + ' : ' +
+        data[0].value.toFixed(2) + '</div>';
+    };
+
+    var metricPrefix = function(maxValue) {
+      var m = {
+        prefix: '',
+        devider: 1
```

Comment:
Also.. you may be able to use `numeral(val).format('0.0 b')` instead of `metricPrefix` ( http://numeraljs.com/#format-bytes )